src/main.cc

```diff
@@ -774,6 +778,16 @@ main(int argc, char **argv)
 	if (strlen(tarantool_path) < strlen(tarantool_bin))
 		panic("executable path is trimmed");
 
+	if (script == NULL && (opt_mask & (O_INTERACTIVE | O_EXECUTE)) == 0) {
+		static const char misuse_msg[] = "Invalid usage:"
+			" please either provide a Lua script name"
+			" or set -i CLI flag to spawn Lua REPL.\n\n"
+			;
```

Comment:
Nit: I would format it like the help message below: with newlines instead of a single long line.

Reply:
OK, no problem. Fixed.